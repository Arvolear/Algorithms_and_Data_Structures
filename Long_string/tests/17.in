14
228016129461280