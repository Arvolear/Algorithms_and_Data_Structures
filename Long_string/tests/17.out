228016129461294
228016129461266
3192225812457920
16286866390091